### ISSUE TITLE
Accessibility] [Screen reader -Playground]: Main landmark is not defined in the page. [Sev2][WCAG1.3.1]

### ISSUE THREAD

**Issue by Laxmidurga** [microsoft/tsdoc.org-website#39]
**Test Environment:**
OS Build: Windows 11
Version: 21H2 (OS Build 22000.318)
Browser: Edge
Version 96.0.1054.43 (Official build) (64-bit)
URL: https://tsdoc.org/
Screen reader: Narrator
Tool: Accessibility insights for web

**Repro Steps:**
1.	Open the above URL and login with valid credentials.
2.	TSDoc page will get displayed.
3.	Now, navigate to the ‘Playground’ link and activate it.
4.	Now, enable landmarks toggle button in AI tool and check the whether the main landmark is defined in the page or not.
5.	Now, turn on narrator and press 'CapsLock+Space' key to enter scan mode.
6.	Now, press landmark shortcut keys 'D' to check for the next landmark and use 'CapsLock+N' to check for the main landmark.

**Actual:**
Main landmark is not defined in the ‘Playground’ page.

**Expected:**
Main landmark should be defined in the ‘playground’ page.

**User Impact:**
Users who depend on assistive technologies like screen readers will use landmarks to navigate the page. So, screen reader users will face difficulty in navigating and understanding the web elements of the web page if landmarks are not defined in the page.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

<img width="960" alt="WCAG1 3 1_Main Landmark is not defined in the playground page" src="https://user-images.githubusercontent.com/88436727/150645169-ed97a639-aa33-4bca-a9b7-1fcd07b6374e.png">
